website/docs/r/app_service_slot_virtual_network_swift_connection.html.markdown

```diff
@@ -79,9 +79,9 @@ resource "azurerm_app_service_slot_virtual_network_swift_connection" "example" {
 
 The following arguments are supported:
 
-* `app_service_id` - (Required) The ID of the App Service to associate to the VNet. Changing this forces a new resource to be created.
+* `app_service_id` - (Required) The ID of the App Service or Function App to associate to the VNet. Changing this forces a new resource to be created.
 
-* `slot_name` - (Required) The name of the App Service Slot. Changing this forces a new resource to be created.
+* `slot_name` - (Required) The name of the App Service Slot or Function App Slot. Changing this forces a new resource to be created.
```

Comment:
whilst these are shared today - longer-term we'll likely want to introduce an alias for the function app resources

this isn't a big deal right now, and these do technically work - but we'll probably look to add aliases for these resources in 3.0 - since (based on Github issues) folks appear to be unaware these are the same thing